torch/autograd/gradcheck.py

```diff
@@ -95,6 +95,7 @@ def get_numerical_jacobian(fn, input, target, eps=1e-3):
 
 
 def get_analytical_jacobian(input, output):
+    input = contiguous(input)
```

Comment:
That's what I'm referring to, not the argument order issue; variable.contiguous() is no longer "inplace".  This can cause non-obvious problems way later in your code (yes, your original code was probably incorrect).

Reply:
ohhhh I see. Still, we didn't make the input contiguous in here before. Is it just a batch fix for a bunch of failing tests or what?

Reply:
Yes, exactly -- the tests failed because the prior call to `get_numerical_jacobian` no longer makes input contiguous "in-place".

Reply:
Sorry, argument order was in https://github.com/pytorch/pytorch/pull/3704/files not this one.